src/lib/db/interface.ts

```diff
@@ -143,4 +156,36 @@ export default class TodoDatabase {
       });
     });
   }
+
+  insertUser(user: User): Promise<boolean> {
```

Comment:
there is `insert` method now.
use that instead.
delete methods `insertUser` and `insertTodo`